manifests/g/Google/Chrome/Canary/98.0.4747.0/Google.Chrome.Canary.installer.yaml

```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: Google.Chrome.Canary
-PackageVersion: 98.0.4746.0
+PackageVersion: 98.0.4747.0
```

Comment:
![image](https://user-images.githubusercontent.com/15158490/144723941-616b00a6-8288-4629-b962-fac7a28cf7cd.png)
Somehow this one also shows .1, prolly changes to .0 later..
```suggestion
PackageVersion: 98.0.4747.1
```